docs/components/table-view.md

```diff
@@ -743,6 +817,99 @@ This slot is only available when the `actions` header key is present in [`header
 </KTableView>
 ```
 
+### bulk-action-items
+
+Slot for passing bulk action dropdown items.
```

Comment:
suggestion: can you add a note about this being required if bulk actions are enabled?